src/xeus_sqlite_interpreter.cpp

```diff
@@ -22,6 +22,14 @@
 
 namespace xeus_sqlite
 {
+
+    std::string parse_code(const std::string& code)
+    {
+        //verifys if user wants to create a new db
```

Comment:
So regarding parse_code I think you should check what magics are in the Jupyter context. 
Magics usually start with `%`

So I think if you search for a first character that is `%` and then parse the rest of the command using your own invented syntax, that would be good.

E.g. 

`%load /path/to/my/database.db`

should load a new database into the context.

I can't come up with other necessary commands right now.

Reply:
Alright, gonna look for it. Thanks :)

Reply:
but you will have to implement this yourself :) so just check the string for `%` at the beginning and then check what the command is, and then do e.g. the loading :) with that we can remove the hard-coded database file.

Reply:
Ohh okay.
I thought it was some magic. :)

Reply:
no magic to implement the magic ;)